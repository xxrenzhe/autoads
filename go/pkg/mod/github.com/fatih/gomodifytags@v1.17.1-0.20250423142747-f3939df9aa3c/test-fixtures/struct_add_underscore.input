package foo

type foo struct {
	foo_bar string
}
