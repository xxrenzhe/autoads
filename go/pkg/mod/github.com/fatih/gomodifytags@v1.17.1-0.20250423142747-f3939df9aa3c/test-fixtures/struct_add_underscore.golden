package foo

type foo struct {
	foo_bar string `json:"foo_bar"`
}
